### PR TITLE
Add variable for setting the additional categories on cluster

### PR DESCRIPTION
**Pull request by thunderboltsid**
The diff also includes a patch to set the additional categories on nutanixmachinetemplate resources for control plane and worker machine deployments.

### CODE REVIEW

File: templates/clusterclass/clusterclass.yaml [hunk 1]
```diff
@@ -37,377 +37,414 @@ spec:
...
-          - type: DiskPressure
-            status: "True"
-            timeout: 300s
-          - type: PIDPressure
-            status: "True"
-            timeout: 300s
-          - type: NetworkUnavailable
-            status: "True"
-            timeout: 300s
+      - class: nutanix-quick-start-worker
```
Comment (deepakm-ntnx) [inline templates/clusterclass/clusterclass.yaml:40]: is this change required?

**Review by deepakm-ntnx — DISMISSED** [review]
pls check the comments before merging

File: templates/clusterclass/clusterclass.yaml (hunk 1 above)
Comment (thunderboltsid) [reply templates/clusterclass/clusterclass.yaml:40]: Good catch, no it isn't. I think my IDE auto indented it.

**Review by dkoshkin — DISMISSED** [review]

**Review by deepakm-ntnx — APPROVED** [review]

**Review by adiantum — APPROVED** [review]
/lgtm
/approve
/meow
